### PR TITLE
Add resource string for the newly added Stop Sensor Confirmation

### PR DESCRIPTION
**Pull request by Navid200**
Adding a resource string for the stop sensor confirmation that was added with build [2021.05.16](https://github.com/NightscoutFoundation/xDrip/releases/tag/2021.05.16) (https://github.com/NightscoutFoundation/xDrip/commit/84e4376c601e52c561ab17a34f691136b62ef27d).

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1496,6 +1496,7 @@
     <string name="__do_you_want_to_change_settings_or_start_sensor">\n\nDo you want to change settings or start sensor?</string>
     <string name="change_settings">Change settings</string>
     <string name="do_you_want_to_delete_and_reset_the_calibrations_for_this_sensor">Do you want to delete and reset the calibrations for this sensor?</string>
+    <string name="sensor_stop_confirm">Do you want to Stop sensor and end the current session?</string>
```
Comment (tolot27) [inline app/src/main/res/values/strings.xml:1499]: Please change it into "Do you want to stop this sensor." IMHO, many users do not understand what "session" means. The term in not introduced, AFAIK.

**Review by tolot27 — CHANGES_REQUESTED** [review]
Please always provide a description and refer the original commit (84e4376 in this case), if appropriate. Tanks!

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Navid200) [reply app/src/main/res/values/strings.xml]: Done

**Comment by Navid200** [comment]
This is an issue that I closed because it is an expected behavior.
https://github.com/NightscoutFoundation/xDrip/issues/1239

There are behaviors that we tell users to accept and put up with until we fix them.
Many such issues have already been addressed.

**Review by tolot27 — APPROVED** [review]
Since it is evident for non-English users to fully understand the consequences of the stop sensor confirmation dialog, it is important to provide a translatable resource string.

**Comment by jamorham** [comment]
Since this string probably is unlikely to change much then this makes sense, in future we can use the `translatable="false"` suggestion for strings where the wording may change during the early stages of a feature.

**Review by jamorham — APPROVED** [review]

**Pull request merged**